packages/react-core/src/components/ClipboardCopy/ClipboardCopy.tsx

```diff
@@ -74,7 +74,7 @@ export interface ClipboardCopyProps extends Omit<React.HTMLProps<HTMLDivElement>
   /** A function that is triggered on changing the text. */
   onChange?: (event: React.FormEvent, text?: string | number) => void;
   /** The text which is copied. */
-  children: React.ReactNode;
+  children: string;
```

Comment:
Would we want to update the onCopy and clipboardCopyFunc params so that `text` is `string | number` similar to the onChange prop while we're at it?

Reply:
could make it `string`, not sure why we have `| number` there. It's always going to be a string even if it is a number.